pkg/sdk/testint/application_roles_gen_integration_test.go

```diff
@@ -78,6 +84,63 @@ func TestInt_ApplicationRoles(t *testing.T) {
 		assertApplicationRoles(t, appRoles, "app_role_2", "some comment2")
 	})
 
+	t.Run("show grants to application - grant to role", func(t *testing.T) {
+		role, cleanupRole := testClientHelper().Role.CreateRole(t)
+		t.Cleanup(cleanupRole)
+
+		id := sdk.NewDatabaseObjectIdentifier(application.Name, "app_role_1")
+		// grant the application role to the role
+		kindOfRole := sdk.NewKindOfRoleRequest().WithRoleName(sdk.Pointer(role.ID()))
+		gr := sdk.NewGrantApplicationRoleRequest(id).WithTo(*kindOfRole)
+		err := client.ApplicationRoles.Grant(ctx, gr)
+		require.NoError(t, err)
+
+		grants, err := client.Grants.Show(ctx, &sdk.ShowGrantOptions{To: &sdk.ShowGrantsTo{Role: role.ID()}})
+		require.NoError(t, err)
+		require.NotEmpty(t, grants)
+		require.NotEmpty(t, grants[0].CreatedOn)
```

Comment:
these 4 lines are repeated in the next test, could we extract the assertion?

Reply:
Done.